### ISSUE TITLE
Inconsistent behavior on foder contents view

### ISSUE THREAD

**Issue by hvelarde** [plone/Products.CMFPlone#1558]
This is the view in the root of the site:

![selection_066](https://cloud.githubusercontent.com/assets/341393/14906796/17fe9280-0d96-11e6-827e-1dd951a2dde9.png)

This is the same view inside a folder; a new widget is shown:

![selection_067](https://cloud.githubusercontent.com/assets/341393/14906802/30c8a2a6-0d96-11e6-8db1-81f34cad9295.png)

When you try to use the widget, it is shown outside the window:

![selection_068](https://cloud.githubusercontent.com/assets/341393/14906806/43640108-0d96-11e6-9477-2afbee821c60.png)

**Comment by davilima6** [comment]
This is a widget for folderish items so I agree it should be also present at root level.

There are other issues:
1. The dropdown menu opens to the left making it impossible to read without zooming out the browser
2. We could remove the menu item "Select all contained items" because there's a duplicated and much more intuitive UI component to perform the same action (checkbox in the 1st column in table header row). Bonus 1: we normalize all items down to the same length. Bonus 2: we solve half of the next issue.
3. The widget simply vanishes if you click "Select all contained items" or just select one of the folder's items and then deselect it. Only (!) way to bring it back is to click the table header checkbox mentioned above.
4. We could make `Edit` come before `Open` in order to separate write operations from read
5. We could rename the "View" title when you hover the "new window" icon to the right of each folder item to "Open item" to make it easier to understand it does the same as the widget "Open" action.
6. We should either choose another icon or make the current open `_blank`; it's misleading now. I don't think that in this case it'd be an a11n problem, do you @polyester?

**Comment by hvelarde** [comment]
@davilima6 the first issue was already reported in #1126

**Comment by thet** [comment]
This current-folder widget should not be present at Plone site root level. You don't want to cut/copy/paste a whole Plone site.

IMO, this widget has some other problems too: It duplicates some functionality already available in the button action bar above and you can achieve the same functionality, if you go a level up and select the folder to operate on.
Because of not optimal UX I removed it in this PR: https://github.com/plone/mockup/pull/651

**Comment by davilima6** [comment]
Indeed, currently half of that widget's actions make no sense for a Plone Site object, as you said the copy/cut/paste ones.

`Edit`, `Select all` and `Open` do work however (the former curiously serving as an alias to `@@site-controlpanel`, maybe should be a more educative redirect) so I'm not sure if we should really remove the widget, thus making the operate on current context story harder (having to go up one level).

**Comment by jensens** [comment]
Closing as outdated (7 years old). About inconsistent folder_contents widgets in old Mockup. The referenced PR #651 was merged and folder_contents was rewritten for Plone 6.
If you think this issue is still relevant please reopen it. Thank you for the original report!
